dataflow/python/bigquery_import.py

```diff
@@ -328,14 +363,21 @@ def run(argv=None):
         write_disposition=beam.io.BigQueryDisposition.WRITE_TRUNCATE,
         create_disposition=beam.io.BigQueryDisposition.CREATE_IF_NEEDED))
 
-# Omit response bodies from the pipeline until we can make better use of sharding.
-#    (hars
-#      | 'MapResponseBodies' >> beam.FlatMap(get_response_bodies)
-#      | 'WriteResponseBodies' >> beam.io.WriteToBigQuery(
-#        get_bigquery_uri(known_args.input, 'response_bodies'),
-#        schema='page:STRING, url:STRING, body:STRING, truncated:BOOLEAN',
-#        write_disposition=beam.io.BigQueryDisposition.WRITE_TRUNCATE,
-#        create_disposition=beam.io.BigQueryDisposition.CREATE_IF_NEEDED))
+    (hars
+      | 'MapResponseBodiesA' >> beam.FlatMap(get_response_bodies_a)
+      | 'WriteResponseBodiesA' >> beam.io.WriteToBigQuery(
+        get_bigquery_uri(known_args.input, 'response_bodies'),
+        schema='page:STRING, url:STRING, body:STRING, truncated:BOOLEAN',
+        write_disposition=beam.io.BigQueryDisposition.WRITE_APPEND,
```

Comment:
Should this be `WRITE_TRUNCATE`? So if need to rerun then existing rows will be removed?

I agree for B it should be `WRITE_APPEND` but not for A.

Reply:
Not sure if A is always guaranteed to write before B since the branches are run in parallel. If we rerun the pipeline for a crawl we should delete the BQ tables first, either manually or as a new step in the `sync_har` script.

Reply:
Ah makes sense. Would be nice if we could automate this to avoid issues in future.